java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -316,6 +318,75 @@ public interface ServerManagementClusterCommands {
      */
     CompletableFuture<ClusterValue<Long>> lastsave(Route route);
 
+    /**
+     * Deletes all the keys of all the existing databases. This command never fails.<br>
+     * The command will be routed to all primary nodes.
+     * <br.
+     * To explicitly specify the flushing mode, use {@link #flushall(FlushOption)}.
```

Comment:
Somewhere we have links, somewhere not.
https://github.com/aws/glide-for-redis/blob/5fbda569de9928219248d3de3b3688cea4ec2ade/java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java#L334
https://github.com/aws/glide-for-redis/blob/5fbda569de9928219248d3de3b3688cea4ec2ade/java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java#L515
Do you want me to add missing links or delete existing (in another PR)?

The default mode is configurable, I wanted to avoid long auxiliary describtions. Can you help me with wording if you want to add this?

Reply:
zrangeWithScores is a command with a different name.  Your example is an overloaded function which is easier to see/find in an IDE.